src/components/NewMessage/NewMessage.vue

```diff
@@ -804,14 +804,8 @@ export default {
 
 .wrapper {
 	position: relative;
-	display: flex;
-	justify-content: center;
 	padding: 12px 0 12px;
```

Comment:
```suggestion
	padding: 12px 0;
```
This as well could be shortened